### ISSUE TITLE
Autoformatting moves object to the next line

### ISSUE THREAD

**Issue by nacyolsa** [detekt/detekt#6223]
## Expected Behavior
Autoformatting should not reformat below line to incorrect state
```
private val deferredMessageQueueSubscription = DeferredMessageQueueSubscription(uri = ServiceBusUri("deferredQueueTestUri"))
```

## Observed Behavior
Autoformatting moves `DeferredMessageQueueSubscription(uri = ServiceBusUri("deferredQueueTestUri"))` to the next line and as the results it looks like this:
```
    private val deferredMessageQueueSubscription = 
    DeferredMessageQueueSubscription(uri = ServiceBusUri("deferredQueueTestUri"))
```
I think it may be related to max line length limit but still I think it should not touch this but throw a warning instead.

Related to `PropertyWrapping`.

## Your Environment
* Version of detekt used: 1.23.0
* Operating System and version: Kotlin 1.8.21

**Comment by atulgpt** [comment]
Hi, @nacyolsa in my case autoformatting is working correctly like below
```
private val deferredMessageQueueSubscription = 
    DeferredMessageQueueSubscription(uri = ServiceBusUri("deferredQueueTestUri"))
```
with an indentation in the second line. 
Is wrong formatting happening consistently in your case?

**Comment by nacyolsa** [comment]
For me in both cases like below:

```
private val deferredMessageQueueSubscription =
DeferredMessageQueueSubscription(uri = ServiceBusUri("deferredQueueTestUri"))

class TestC {
    private val deferredMessageQueueSubscription =
    DeferredMessageQueueSubscription(uri = ServiceBusUri("deferredQueueTestUri"))
}
```
Formatting is incorrect.

>Is wrong formatting happening consistently in your case?

For this case always when I run Detekt.
I checked this also in smaller project and it didn't occur so it is connected to something else.

**Comment by atulgpt** [comment]
Hi, @nacyolsa can you also share your `detekt.yml`?

**Comment by nacyolsa** [comment]
@atulgpt Here you go:

```yaml
formatting:
    excludes: [ '**/build/**' ]
    MaximumLineLength:
        active: false
    NoBlankLineBeforeRbrace:
        active: false
    Indentation:
        active: false
    ArgumentListWrapping:
        active: false
    MultiLineIfElse:
        active: false
style:
    excludes: [ '**/build/**' ]
    MagicNumber:
        active: true
        excludes: [ '**/build.gradle.kts', '**/test/**', '**/build/**' ]
    MaxLineLength:
        maxLineLength: 180
        active: true
    ForbiddenComment:
        active: true
    RedundantHigherOrderMapUsage:
        active: false
    VarCouldBeVal:
        active: false
naming:
    excludes: [ '**/build/**' ]
    MemberNameEqualsClassName:
        active: true
        excludes: [ '**/buildSrc/**' ]
    InvalidPackageDeclaration:
        active: false
    NoNameShadowing:
        active: false
    TopLevelPropertyNaming:
        active: false
complexity:
    excludes: [ '**/build/**' ]
    LongParameterList:
        functionThreshold: 15
        excludes: ['**/test/**']
    TooManyFunctions:
        thresholdInFiles: 15
        thresholdInClasses: 15
coroutines:
    InjectDispatcher:
        active: false

ArrowRuleSet:
    NoEffectScopeBindableValueAsStatement:
        active: true
```

**Comment by atulgpt** [comment]
Hi, @nacyolsa actually it is an issue(or intended behaviour) with ktlint. Actually when you run `PropertyWrapping` rule(`PropertyWrapping` is a stable rule hence it is enabled by default) without `Indentation` then `PropertyWrapping` simply inserts the new line and doesn't indent it.

Either you have to enable the `Indentation` or disable the `PropertyWrapping` rule.

Can you also create an issue at the ktlint repo? In my opinion, this can be fixed. @paul-dingemans might be able to answer there

**Comment by nacyolsa** [comment]
Issue in ktlint https://github.com/pinterest/ktlint/issues/2095

**Comment by 3flex** [comment]
@nacyolsa I've updated ktlint to 0.50.0. Can you please try a detekt snapshot and check that the upstream fix has resolved the issue?

**Comment by nacyolsa** [comment]
@3flex I can check it but I can't find snapshot name in this build https://github.com/detekt/detekt/actions/runs/5450089240
What is its name?

**Comment by 3flex** [comment]
Please follow these instructions: https://detekt.dev/docs/introduction/snapshots

**Comment by nacyolsa** [comment]
I tried with `main-SNAPSHOT`  it downloaded `detekt-gradle-plugin-main-20230704.022131-1376` version.

Looks like issue is fixed. Results:
```
    private val deferredMessageQueueSubscription = 
        DeferredMessageQueueSubscription(uri = ServiceBusUri("deferredQueueTestUri"))
 ```

Instead of

```
    private val deferredMessageQueueSubscription = 
    DeferredMessageQueueSubscription(uri = ServiceBusUri("deferredQueueTestUri"))
```
